news/2017-06-27-beatmap-spotlight-may-2017.md

```diff
@@ -26,96 +27,98 @@ First of all, congratulations to the winners of the April 2017 Spotlights. They
+
+He did a good job on highlighting certain beats magnitude with a suitable slider velocity. By advancing through the map, player could easily experience the consciousness of the music along with the note arrangement emphasized by the mapper intention. Have a try while you are on it.
 
 [![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/shine.jpg)](https://osu.ppy.sh/s/456289)
 
-For his 5th ranked map, Frim4503 offers us another 5K marathon, this time for a drum & bass love live opening remix !
-As usual we have a mix of LN and chords playing really good, making it a really fun map to try big accuracy scores. SV are really lightly used aswell with a quite unique style, focussing on the voice before each kiai. Without changing much of music/mapping style, Frim4503 show us one more time that anti-meta is not dead with a really accessible and catchy map.
+For his 5th ranked map, **[Frim4503](https://osu.ppy.sh/users/2945775)** offers us another 5K marathon, this time for a drum & bass love live opening remix!
+As usual, the usage of long notes and chords play really well, making it a really fun map to try big accuracy scores. Scroll speed changes are lightly used with a quite unique style, focus on the voice before each kiai time. Without changing much of music/mapping style, he shows us one more time that anti-meta is not dead with a really accessible and catchy map.
 
 [![](/wiki/shared/news/2017-06-27-beatmap-spotlights-may-2017/doppelganger.jpg)](https://osu.ppy.sh/s/407153)
 
-Apart from having the hardest difficulty ever ranked in osu!mania, it counts with a really enjoyable and challenging patterns in every single difficulty. We also have a few GDs coming from Evening, jakads and Lude. Evening's chart is not really heavily charted, but watch out the svs! They're way more noticeable in this difficulty than in the other ones. Every single mapper and modder has done a really good job checking the pattern's quality and playability of each difficulty (even though alter ego it's hell itself, someone with a good eye can really see how good the patterns fits with the song).
+While this set is being a hard one in osu!mania, it comes with a really enjoyable and challenging patterns in every single difficulty. We also have a few guest difficulties coming from **[Evening](https://osu.ppy.sh/users/2193881)**, **[jakads](https://osu.ppy.sh/users/259972)** and **[Lude](https://osu.ppy.sh/users/6974536)**. Evening's chart is not really overtly heavy, but watch out for the scroll speed changes! They're way more noticeable in this difficulty than in the other ones. Every single mapper and modder has done a really good job checking the pattern's quality and playability of each difficulty (even though Alter Ego is hell itself, someone with a good eye can really see how good the patterns fits with the song).
```

Comment:
While this set may be harder than others in the chart so far, its challenge is no detterent to its enjoyability, sporting some difficult but rewarding patterning in every difficulty. Guest difficulties from Evening, jakads and Lude provide some variety within the set overall, with Evening's chart in particular being somewhat light, but chock full of scroll speed changes.

Alter Ego is definitely one of the harder songs around to map, but the people behind the set have done an excellent job of patterning their charts to fit it, and it is definitely something you should try and see for yourself.